www/main/common/utils/convert-to-camel-case.js

```diff
@@ -0,0 +1,9 @@
+const convertToCamelCase = (str, isPascalCase = false) => {
```

Comment:
can we use https://lodash.com/docs/4.17.15#camelCase ?

Reply:
okay